_posts/2021-11-29-plugins-intro.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Introduction to OpenSearch Plugins"
+authors: 
+  - vemsarat
+  - kazabdu
+date: 2021-11-22
+categories:
+  - technical-post
+twittercard:
+  description: "This post walks through how plugins work in OpenSearch and how a custom Java Security Manager handle the security of OpenSearch."
```

Comment:
handleS

also maybe JSM doesn't "handle the security" and it's kind-a odd to mention it here so maybe just "This post contains an overview of how OpenSearch plugins are loaded and interact with Java Security Manager."